services/QuillLMS/spec/workers/sync_vitally_worker_spec.rb

```diff
@@ -8,12 +8,39 @@
   describe "#perform" do
     it "make queries for schools and users and enqueue them for further jobs" do
       ENV['SYNC_TO_VITALLY'] = 'true'
```

Comment:
All the `ENV['SYNC_TO_VITALLY']` calls in these tests should be removed since they can leak state.
I would opt a single  `before { stub_const('ENV', {'SYNC_TO_VITALLY' => 'true'}) }` that applies to all of the specs instead.